CODE_OF_CONDUCT.md

```diff
@@ -8,7 +8,9 @@ Examples of unacceptable behavior by participants include the use of sexual lang
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. Project maintainers who do not follow the Code of Conduct may be removed from the project team.
 
-This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
+This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.  An individual is deemed to be representing the project or its community if discussing topics germane to the project, or when the individual has explicitly identified as belonging to the project or community in the course of the conversation.
+
+This code of conduct does not apply to opinions expressed in public fora when not representing the project or its community.
```

Comment:
Fora is a good word here (and 100% accurate) but it *is* a little obscure. Perhaps a more plainly spoken phrasing would help maintain the voice?